script/libtest.bash

```diff
@@ -329,10 +329,17 @@ test_kubeless_function_update() {
     update_function $func
     verify_function $func ${func}-update-verify
 }
+create_basic_auth_secret() {
+    sudo apt-get install apache2-utils
```

Comment:
I would move this to the .travis.yaml file along with the others things to install